.github/workflows/build_and_test.yml

```diff
@@ -136,7 +136,7 @@ jobs:
 
     - name: Test docs
       run: |
-        sphinx-build -n -v -E -W ./docs ./tmp/ert_docs
+        sphinx-build -n -v -E -W ./docs/ert ./tmp/ert_docs
```

Comment:
Are there a similar test for everest docs?

Reply:
Yes, it is in the everest workflow, renamed it to `test_everest.yml`